src/db_options.rs

```diff
@@ -2664,6 +2666,28 @@ impl Options {
         }
     }
 
+    /// StatsLevel can be used to reduce statistics overhead by skipping certain
+    /// types of stats in the stats collection process.
+    pub fn set_statistics_level(&self, level: StatsLevel) {
+        unsafe { ffi::rocksdb_options_set_statistics_level(self.inner, level as c_int) }
+    }
+
+    pub fn get_ticker_count(&self, ticker: Ticker) -> u64 {
```

Comment:
Add some description to the public methods, please.

Reply:
Original rocksdb functions don't have documentation but I've put something there to explain.